cli/src/main/resources/nls/ide_en.properties

```diff
@@ -0,0 +1,26 @@
+usage=Usage:
```

Comment:
I do not see that we ever want to support different texts for `en` vs. `en_US`.
The `ide.properties` will be used as fallback for any language that is not supported more explicitly.
We could add `ide_es.properties` and `ide_fr.properties` some day in the future...

Reply:
Done by removing